pages/en-us/components/footer.mdx

```diff
@@ -0,0 +1,190 @@
+import { Playground, ExampleBlock, Attributes, Layout } from 'lib/components'
+import { Footer, Text } from 'components'
+
+export const meta = {
+  title: 'footer',
+  group: 'Surfaces',
+}
+
+## Footer
+
+<Playground
+  scope={{ Footer }}
+  code={`
+<Footer>
+  <Footer.Group title="Company">
+    <Footer.Link href="#">Home</Footer.Link>
```

Comment:
Maybe we should allow this:
```ts
<Footer.Group title="Company">
   <Footer.Item href="#">Home</Footer.Item>
   <Footer.Item><Link href="" target="" rel=""></Link></Footer.Item>
   <Footer.Item>
     <NextLink href="" passHerf>
       <Link  target="" rel=""></Link>
     </NextLink>
  </Footer.Item>
</Footer.Group>
```
Here component `Footer.Item` should only have style constraints, we need to consider all scenarios as much as possible.

Reply:
In addition, we may also need to make a distinction between `Link` and `Text`, there are scenarios where the user does not need the link, but only as a display or hover tip, etc.

```ts
<Footer.Item>
  <Popover><Text></Text></Popover>
</Footer.Item>
```